fiddler/README.md

```diff
@@ -0,0 +1,64 @@
+# Agent Check: Fiddler
+
+## Overview
+
+This check monitors AI model performance using [Fiddler][1] through the Datadog Agent.
```

Comment:
Could you include more information on what this does and the value of the offering?